eclipse-scout-core/src/modeselector/ModeSelector.js

```diff
@@ -57,6 +63,11 @@ export default class ModeSelector extends Widget {
     }, this);
   }
 
+  _renderSlider() {
+    this.$slider = this.$container.appendDiv('slider');
```

Comment:
Maybe call css class mode-slider to avoid a conflict with the slider from slider.less